packages/core/nuxt-theme-module/theme/helpers/errors/index.js

```diff
@@ -0,0 +1,5 @@
+import { authErrors } from './authentication';
```

Comment:
if you're writing a function it shouldn't be named with a verb (function is doing something and name should tell what its doing), suggestion above (getFriendlyErrors) with a new signature